.github/workflows/ci.yml

```diff
@@ -63,8 +72,14 @@ jobs:
           - windows-latest
     steps:
       - uses: actions/checkout@v4
+
+      - name: Setup Rust toolchain
+        uses: ./.github/actions/setup-builder
+        with:
+          rust-version: ${{ env.rust_msrv }}
+
       - name: Build
-        run: cargo build
+        run: make build
```

Comment:
I don't get this. It seems we only installed stable toolchain `rust_msrv`, but not used in the build process. :eyes:

Reply:
We should `cargo +${{ env.rust_msrv}} build`, or `rustup override set ${{ env.rust_msrv}}`